### PR TITLE
[v12] Distroless doc updates

### PR DESCRIPTION
**Pull request by tcsc**
Documentation updates making "distroless" the default image

### CODE REVIEW

**Review by stevenGravy — CHANGES_REQUESTED** [review]
Make work with existing docker installation https://goteleport.com/docs/management/guides/docker/.  This currently fails

**Comment by ptgott** [comment]
Just to double-check, does this need to be forward ported to `master` (or was there already a PR for `master`?)

**Review by r0mant — APPROVED** [review]
lgtm once Steven's comment about something not working is addressed.

**Review by ptgott — CHANGES_REQUESTED** [review]
I could have sworn I left a review of this earlier, but I'm not seeing it in the PR conversation. In any case, the Markdown tables are formatted incorrectly and do not render as tables. We should make sure this looks okay on the docs dev server before merging.

File: docs/pages/includes/docker-images-oss.mdx
Comment (ptgott) [inline docs/pages/includes/docker-images-oss.mdx:22]:
```suggestion
The `*-debug` images are intended for troubleshooting deployments, not for production use. These include a shell and BusyBox to help you investigate issues.
```

File: docs/pages/includes/enterprise/docker-images.mdx
Comment (ptgott) [inline docs/pages/includes/enterprise/docker-images.mdx:26]:
```suggestion
The `*-debug` images are intended for troubleshooting deployments, not for production use. These include a shell and BusyBox to help you investigate issues.
```

**Comment by tcsc** [comment]
> I could have sworn I left a review of this earlier

The review you're thinking of was on `master`. This is a backport. Fixed in both.

**Review by ptgott — APPROVED** [review]

**Review by stevenGravy — APPROVED** [review]
lgtm

**Pull request merged**
